test/Proxy.test.js

```diff
@@ -62,6 +63,11 @@ function startProxyServers() {
 }
 
 describe('Proxy', () => {
+  // issue: https://github.com/webpack/webpack-dev-server/pull/1633#issuecomment-463938675
+  if (shouldSkipTestSuite()) {
+    return;
+  }
```

Comment:
All tests failed? Maybe we can skip only one test?

Reply:
As far as I confirm, running two tests seems to fail.

Reply:
@hiroppy can we ship only their, thanks?

Reply:
The first test(anyone) will succeed but subsequent tests will fail. 
Probably postprocessing would be a problem.

Once we merge this PR and then I think that it is better to investigate the cause separately.(maybe difficult)

Reply:
:+1: Let's do it